build.gradle

```diff
@@ -353,6 +354,8 @@ dependencies {
   implementation group: 'org.springframework', name: 'spring-tx', version: versions.springVersion
   implementation group: 'org.springframework', name: 'spring-web', version: versions.springVersion
   implementation group: 'org.springframework', name: 'spring-webmvc', version: versions.springVersion
+  
+  implementation group: 'org.bouncycastle', name: 'bcpkix-jdk15on', version: '1.70'
```

Comment:
Can you please upgrade the bouncycastle with bcpkix-jdk18on

Reply:
It should be bcpkix-jdk17on

Reply:
It should be bcpkix-jdk17on